internal/database/dbconn/new.go

```diff
@@ -19,6 +20,10 @@ type Opts struct {
 	// AppName overrides the application_name in the DSN. This separate parameter is needed
 	// because we have multiple apps connecting to the same database, but have a single shared DSN configured.
 	AppName string
+
+	// Databases is set of migration specs that should be executed on the fresh connection if the database
+	// appears to be out-of-date.
+	Databases []*Database
```

Comment:
```suggestion
	DatabasesToMigrate []*Database
```
? 

This seems kinda unclear to me from just reading the callsites otherwise. After all, I mostly ask the `New` function to create a new connection. This field causing migrations to run as a side-effect and the name not reflecting it feels incorrect to me.

Reply:
I was doing very small chunks of work, this was to unblock a refactor of that type completely, but I'll make a more meaningful name just for the diff.

Reply:
No need to do additional work here when you already got plans to get rid of it anyways. It doesn't bother me that much.

Reply:
Renamed anyway